packages/playground/website/src/components/log-modal/index.tsx

```diff
@@ -29,9 +29,13 @@ export function LogModal(props: { description?: JSX.Element; title?: string }) {
 			)
 			.reverse()
 			.map((log, index) => (
-				<pre className={css.logModalLog} key={index}>
-					{log}
-				</pre>
+				<div
+					className={css.logModalLog}
+					key={index}
+					dangerouslySetInnerHTML={{
+						__html: log.replace(/Error:/, '<mark>$&</mark>'),
```

Comment:
Nice :slightly_smiling_face:

Reply:
PHP errors start with Fatal, so it would be nice to include it as an option. 
You can test it in this example http://localhost:5400/website-server/#{%20%22landingPage%22:%20%22/%22,%20%22steps%22:%20[%20{%20%22step%22:%20%22runPHP%22,%20%22code%22:%20%22%3C?php%20thrown%20new%20Exception('test');%20?%3E%22%20}%20]%20}

```suggestion
						__html: log.replace(/Error:|Fatal:/, '<mark>$&</mark>'),
```